src/main/java/seedu/address/model/interview/Interview.java

```diff
@@ -40,7 +40,7 @@ public Interview(Applicant app, String role, String timing, boolean isDone) {
 
     /**
      * Returns true if both Interviews have the same Applicant & Timing or if both Interviews are the same object
-     * Adapted from AB3's Person.isSamePerson() method
+     * Adapted from AB3's Applicant.isSameApplicant() method
```

Comment:
I think the comment only makes sense with the original 'AB3's Person.isSamePerson() method, could possibly be reverted?